openspec/changes/extract-shared-base-cache-code/specs/shared-expiry-helpers/spec.md

```
@@ -0,0 +1,32 @@
+## ADDED Requirements
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)